### PR TITLE
Black 19.10b on misc files

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses issue #2008, largely avoiding files already updated with black 19.03b which is slightly different, some more problematic cases (e.g. dictionaries of data), and a few files likely to be updated in separate work.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
Strange, GitHub shows TravisCI as pending, but it finished OK (apart from ARM64 which is allowed to fail).

File: Bio/SeqIO/SeqXmlIO.py [hunk 1]
```diff
@@ -262,9 +283,13 @@ def startDBRefElement(self, attrs):
                 elif localname == "id":
                     ID = value
                 else:
-                    raise ValueError("Unexpected attribute '%s' found for DBRef element", key)
+                    raise ValueError(
+                        "Unexpected attribute '%s' found for DBRef element", key
```
Comment (MarkusPiotrowski) [inline Bio/SeqIO/SeqXmlIO.py]: Shouldn't this be `% key` instead of `, key`?

File: Bio/SeqIO/SeqXmlIO.py [hunk 2]
```diff
@@ -305,9 +336,13 @@ def startPropertyElement(self, attrs):
                 elif localname == "value":
                     property_value = value
                 else:
-                    raise ValueError("Unexpected attribute '%s' found for property element", key)
+                    raise ValueError(
+                        "Unexpected attribute '%s' found for property element", key
```
Comment (MarkusPiotrowski) [inline Bio/SeqIO/SeqXmlIO.py]: Also here, `% key`?

Comment (peterjc) [reply Bio/SeqIO/SeqXmlIO.py]: Good catch, yes - a lurking error.

File: Bio/SeqIO/SeqXmlIO.py (hunk 1 above)
Comment (peterjc) [reply Bio/SeqIO/SeqXmlIO.py]: Yes, good catch

File: Bio/Alphabet/__init__.py [hunk 3]
```diff
@@ -242,8 +265,9 @@ def contains(self, other):
         hierarchy, and attempts to check the gap character.  This fails
         if the other alphabet does not have a gap character!
         """
-        return (other.gap_char == self.gap_char and
-                self.alphabet.contains(other.alphabet))
+        return other.gap_char == self.gap_char and self.alphabet.contains(
+            other.alphabet
+        )
```
[the remark this answers is not in the document]
Comment (peterjc) [reply Bio/Alphabet/__init__.py]: I agree with you on these examples, but don't care enough to disable black here. We're planning to remove/replace the alphabet code this year anyway, and I especially dislike the ``AlphabetEncoder`` part, so we'll deal with it one way or another.

[the remark this answers is not in the document]
Comment (MarkusPiotrowski) [reply Bio/Alphabet/__init__.py]: I just wondered if there is a simple way to force `black` to line-break between the conditions.
I agree that switching `black` off for these small pieces would be cumbersome.

**Comment by MarkusPiotrowski** [comment]
Strange, appveyor fails with a test that should be fine (it passed on Travis). It seems that  `test_CodonTable.py` was not updated in the appveyor environment? Maybe a problem with simultaneously merging #2530?

**Comment by peterjc** [comment]
Let's try re-starting the AppVeyor in case that sorts itself out...

**Comment by peterjc** [comment]
Good news: AppVeyor is happy now

Bad news: merge conflict, it looks like I got the Codon Table stuff on this branch too somehow. I'll take another look.

**Comment by peterjc** [comment]
Will have to rebase again if #2538 gets merged first as that also changes the BLAST files.

**Pull request merged**